### DISCUSSION TITLE
Notifications tab to have a group by Organisation

### DISCUSSION THREAD

**Discussion by IPWright83** [community/community#109555] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great if the notifications page https://github.com/notifications had the ability to group by Organisation.

I've got a single GitHub user account that I use for both work and personal, and I don't want to have to manually filter by eye whether something is relevant or not. While I'm working, I only want to be able to see my work notifications - offering a group by would allow me to do this really easily, while ignoring the other organisations/personal items on the list.

There is a similar discussion elsewhere, but that is specific to the mobile app. I'm proposing this more globally (e.g. I'd like it just on the webpage).

**Comment by IPWright83** [comment]
Interestingly going back and look at the Mobile app, I hadn't realised there was a `Filter` option. I've never noticed it before 🤦🏻‍♂️ However I still think it'd be nice to group :)

**Reply by hangshan-boop** [reply]
you use mobile browser that it is faster than computer browser. Every day, I handle file of mobile browser.
